run-dljc.sh

```diff
@@ -100,179 +116,107 @@ fi
 
```

Comment:
Immediately above, I don't think you need that code.  USER should already be set by the shell.

Reply:
This is a mistake - the variable should have been named something other than `USER`. I forgot that that's the shell built-in.